loadgen/bindings/python_api.cc

```diff
@@ -0,0 +1,182 @@
+#ifndef PYTHON_BINDINGS_H
+#define PYTHON_BINDINGS_H
+
+#include <functional>
+
+#include "third_party/pybind/include/pybind11/functional.h"
```

Comment:
third_party/pybind/include/ should be removed and instead passed in via -I from the compiler command line.

Reply:
Done.